src/helpers/sort-sizes-by-asc-order.ts

```diff
@@ -0,0 +1,9 @@
+const sortOrder = ["XS", "S", "M", "L", "XL"];
+
+function sortSizesByAscOrder<T extends number | string>(items: T[]): T[] {
+	return items.slice().sort((a, b) => {
+		return sortOrder.indexOf(a.toString()) - sortOrder.indexOf(b.toString());
+	});
+}
+
+export { sortSizesByAscOrder };
```

Comment:
I saw the same changes in PR 38, you doest create branched from develop. its very very bad